tox.ini

```diff
@@ -1,7 +1,7 @@
 [tox]
-envlist=py,docs
+envlist=py{27,34},docs
 
-[testenv:py]
+[testenv]
```

Comment:
I believe the previous limitation to the environment would prevent the dependency download for the docs built?

Reply:
It should only fall back to the default env for keys that aren’t defined. The docs env has its own deps and commands so that shouldn’t be an issue. Generative envs make it hard to work with specific testenv blocks, because you have to match the full env name (i.e. py doesn’t count as a match for py27 or py34 so the tests don’t run). You can use conditional settings to avoid unwanted inheritance but I didn’t feel it was needed here.